src/Refund.cs

```diff
@@ -6,14 +6,14 @@ public class Refund : Entity
     {
         public string PaymentId;
 
-        public Refund Fetch(string id)
+        new public Refund Fetch(string id)

```

Comment:
now we can directly fetch a refund using id, we don't need payment_id, how did we supported that in other sdks. Add that here as well